ci: test (almalinux_8_appstream_1.16, arm64v8) faild

```bash
...
8: Pulling from arm64v8/almalinux
no matching manifest for linux/amd64 in the manifest list entries
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -26,8 +26,8 @@ jobs:
           - almalinux_9
           - almalinux_9_appstream_1.22
         arch:
-          - amd64
-          - arm64v8
+          - linux/amd64
+          - linux/arm64/v8
 
     env:
       BUILD_IMAGE: ${{ matrix.image }}

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -47,7 +47,7 @@ jobs:
           ./build -p ${{ matrix.arch }} $(echo ${BUILD_IMAGE} | sed 's/_/:/g')
 
       - name: Remove conflict files
-        if: matrix.arch != 'amd64'
+        if: matrix.arch != 'linux/amd64'
         run: |
           rm -f rpmbuild/SRPMS/*.rpm
           ls -al rpmbuild/SRPMS/

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -27,8 +27,8 @@ jobs:
           - almalinux_9
           - almalinux_9_appstream_1.22
         arch:
-          - amd64
-          - arm64v8
+          - linux/amd64
+          - linux/arm64/v8
 
     env:
       BUILD_IMAGE: ${{ matrix.image }}

--- a/.github/workflows/test.yaml
+++ b/.github/workflows/test.yaml
@@ -47,8 +47,11 @@ jobs:
         run: |
           ./build -p ${{ matrix.arch }} $(echo ${BUILD_IMAGE} | sed 's/_/:/g')
 
+      - name: Normalize artifact name
+        run: echo "ARTIFACT_NAME=$(echo ${{ matrix.image }}-${{ matrix.arch }} | sed 's/\//-/g')" >> $GITHUB_ENV
+
       - name: Upload artifacts
         uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
         with:
-          name: ${{ matrix.image }}-${{ matrix.arch }}
+          name: ${{ env.ARTIFACT_NAME }}
           path: rpmbuild/**/*.rpm

--- a/README.md
+++ b/README.md
@@ -66,9 +66,9 @@ Usage:
         # EPEL 8 Modularity was going away on February 15, 2023
         build almalinux:8:epel-modular:mainline
 
-    Build for RHEL/AlmaLinux/Rocky Linux 9 arm64v8(aarch64) + AppStream module:
-        build -p arm64v8 almalinux:9
-        build -p arm64v8 almalinux:9:appstream:1.22
+    Build for RHEL/AlmaLinux/Rocky Linux 9 linux/arm64/v8(aarch64) + AppStream module:
+        build -p linux/arm64/v8 almalinux:9
+        build -p linux/arm64/v8 almalinux:9:appstream:1.22
 ```
 
 ## Build RPM Packages with Docker
@@ -87,8 +87,8 @@ You can build RPM packages in Docker.
 BUILD_HOSTNAME=el9.example.org ./build -d almalinux:9
 /pkg/build-rpm /pkg/rpmbuild nginx-module-fancyindex.spec
 
-# el9 + arm64v8 + debug shell
-BUILD_HOSTNAME=el9.example.org ./build -d -p arm64v8 almalinux:9
+# el9 + linux/arm64/v8 + debug shell
+BUILD_HOSTNAME=el9.example.org ./build -d -p linux/arm64/v8 almalinux:9
 /pkg/build-rpm /pkg/rpmbuild nginx-module-fancyindex.spec
 
 # el8 + Modular package version + debug shell

--- a/build
+++ b/build
@@ -44,9 +44,9 @@ Usage:
         # EPEL 8 Modularity was going away on February 15, 2023
         ${PACKAGER} almalinux:8:epel-modular:mainline
 
-    Build for RHEL/AlmaLinux/Rocky Linux 9 arm64v8(aarch64) + AppStream module:
-        ${PACKAGER} -p arm64v8 almalinux:9
-        ${PACKAGER} -p arm64v8 almalinux:9:appstream:1.22
+    Build for RHEL/AlmaLinux/Rocky Linux 9 linux/arm64/v8(aarch64) + AppStream module:
+        ${PACKAGER} -p linux/arm64/v8 almalinux:9
+        ${PACKAGER} -p linux/arm64/v8 almalinux:9:appstream:1.22
 
 _EOF_
 }
@@ -80,11 +80,11 @@ build() {
     if [[ ! -z "${PLATFORM}" ]]; then
         # Enable multiarch/qemu-user-static
         docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
-        BUILD_IMAGE="${PLATFORM}/${BUILD_IMAGE}"
+        DOCKER_OPTIONS="--platform ${PLATFORM}"
     fi
 
     # Update build image
-    docker pull "${BUILD_IMAGE}"
+    docker pull ${DOCKER_OPTIONS} "${BUILD_IMAGE}"
 
     # Debug mode
     if [[ "${DEBUG}" = "yes" ]]; then
@@ -92,7 +92,7 @@ build() {
         echo "# Manual Build"
         echo "${BUILD_CMD}"
         echo
-        DOCKER_OPTIONS="-it"
+        DOCKER_OPTIONS="${DOCKER_OPTIONS} -it"
         BUILD_CMD="/bin/bash"
     fi
 

